ERCS/erc-7715.md

```diff
@@ -312,7 +297,7 @@ type RateLimitPolicy = {
 };
 
 // The amount of Native Token that can be spent by a permission
-type NativeTokenSpendLimitPolicy = {
+type NativeTokenSpendLimitPermission = {
```

Comment:
Now, when it is a permission, what is the difference b/w this one and the first one, `NativeTokenTransferPermission` ?

Reply:
good catch, just a duplicate in this case. removed